packages/matrix/tests/registration-with-token.spec.ts

```diff
@@ -174,13 +174,6 @@ test.describe('User Registration w/ Token - isolated realm server', () => {
     let newRealmURL = new URL('user1/personal/', serverIndexUrl).href;
     await enterWorkspace(page, "Test User's Workspace");
 
-    // assert back button brings you back to workspace chooser
-    await page.goBack();
-    await expect(
-      page.locator(`[data-test-workspace="Test User's Workspace"]`),
-    ).toHaveCount(1);
-    await enterWorkspace(page, "Test User's Workspace");
```

Comment:
This was failing in CI _only_, you can see a trace [here](https://boxel-matrix-playwright-reports.stack.cards/host/updateCodePath-redirects-cs-8833/20250611_185220Z/trace/index.html?trace=https://boxel-matrix-playwright-reports.stack.cards/host/updateCodePath-redirects-cs-8833/20250611_185220Z/data/97a5868a67e194d4354bd1085c1596ba062ad654.zip) (while it lasts): pressing the back button was not returning to the workspace chooser. Who knows the underlying cause, but I added a separate test that specifically exercises the back button, and it works in CI 🙃